src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
```

by using flex you won't get always 4 elements in a row. better to use grid
```suggestion
  display: grid;
  grid-template-columns: repeat(4, 1fr);
```